dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6355,10 +6348,11 @@ public String toString() {
       stringBuilder(Name.FUSE_MOUNT_ALLUXIO_PATH)
           .setAlias(Name.WORKER_FUSE_MOUNT_ALLUXIO_PATH)
           .setDefaultValue("/")
+          // TODO(Yichuan): fix the description
           .setDescription(format("The Alluxio path to mount to the given "
-              + "Fuse mount point configured by %s in the worker when %s is enabled "
```

Comment:
Can I just fix the description like this? @LuQQiu

Reply:
please remove the "in the worker"
now it's not related to worker anymore